spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -254,4 +274,11 @@ def check_count(resource, method, expected)
   def check_counts(resource, resources)
     resources.each_pair { |key, value| check_count(resource, key, value) }
   end
+
+  def with_vcr(suffix = nil, &block)
+    path = described_class.name.dup
+    path << "::#{suffix}" if suffix
+
+    VCR.use_cassette(path.underscore, &block)
+  end
```

Comment:
I've added a method like this in a few places, would be nice to add this to the core ems_refresh_helper also